data-common.json

```diff
@@ -80,6 +80,11 @@
       "val": false,
       "note_id": "firefox-private-class-fields",
       "note_html": "The feature have to be enabled via <code>javascript.options.experimental.fields</code> setting under <code>about:config</code>. Private fields are supported by parser, but behave as public fields."
+    },
+    "weakrefs": {
```

Comment:
Since this note is just for this feature, how about defining this locally as the Chrome note?